ProcessManager.h

```diff
@@ -48,5 +49,5 @@ class ProcessManager
 	void PrintContent();
 	char* Read();
 	void getContent(char * Module = NULL);
+	vector<long> scanProcess(long target);
```

Comment:
(int target)
also add scanProcess(long target)
scanProcess(string target)
scanProcess(char target)
only a minor change is required in scanProcess(int target) to generate above mentioned functions

Reply:
done